azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/DeployMojo.java

```diff
@@ -166,6 +152,30 @@ protected void configureAppSettings(final Consumer<Map> withAppSettings, final M
 
     //endregion
 
+    protected FunctionRuntimeHandler getFunctionRuntimeHandler() throws MojoExecutionException, AzureAuthFailureException {
+        final FunctionRuntimeHandler.Builder<?> builder;
+        switch (this.getRuntimeOs()) {
```

Comment:
remove getRuntimeOs but add a function here to get os from runtime

Reply:
Done